doc/install.rst

```diff
@@ -188,6 +188,19 @@ dependencies (numpy, scipy) that scikit-learn requires.
 The following is an incomplete list of OS and python distributions
 that provide their own version of scikit-learn.
 
+Alpine Linux
+----------
+
+Alpine Linux's package is provided through the `official repositories
+<https://pkgs.alpinelinux.org/packages?name=py3-scikit-learn>`_ as
```

Comment:
To fix the [warnings](https://output.circle-artifacts.com/output/job/a85bb54d-9ab6-49de-bcff-2da106b89f0c/artifacts/0/doc/_changed.html) from the CI:

```suggestion
<https://pkgs.alpinelinux.org/packages?name=py3-scikit-learn>`__ as
```

(This makes the reference anonymous to avoid the warning)